system/templates/rolls/preroll-dialog.hbs

```diff
@@ -0,0 +1,71 @@
+{{#*inline "statDescription"}}
+{{#if (eq Using.length 1)}}
+roll +{{lookup Using 0}}
+{{else}}
+roll {{Method}} of {{join Using delimiter=', '}} {{!-- TODO: i18n --}}
+{{/if}}
+{{/inline}}
+
+<form class="preroll-dialog">
+  {{#if move}}
+  <div>{{{enrichMarkdown move.data.data.Trigger.Text}}}</div>
+
+  {{#if move.data.data.Trigger.Options}}
+  <ul>
+    {{#each move.data.data.Trigger.Options}}
+    {{#if Text}}
+    <li>
+      {{ Text }}: <strong>{{>statDescription}}</strong>
+    </li>
+    {{/if}}
+    {{/each}}
+  </ul>
+  {{/if}}
+  {{/if}}
+
+  <div class="dice-roll ironsworn-roll">
+    {{{graphic}}}
+  </div>
+
+  {{#if action}}
+  <div class="form-group">
+    <label>{{localize 'IRONSWORN.Adds'}}</label>
```

Comment:
label not currently associated w/ input

Reply:
Foundry norms. This is how it's meant to look:

![CleanShot 2022-08-19 at 16 37 40](https://user-images.githubusercontent.com/39902/185719647-1a3ae594-d730-4219-8fa9-2cbf2e2879f2.jpg)

And this is how it looks if I move the `<input>` inside the `<label>`:


![CleanShot 2022-08-19 at 16 37 29](https://user-images.githubusercontent.com/39902/185719665-92528a86-463e-4113-85f0-b4302ed9b3f6.jpg)

Trying to avoid inventing a thing that looks like Foundry's form classes without using Foundry's form classes. Let's revisit in a future PR.